src/TerrainGeneration/land_generator.hpp

```diff
@@ -75,7 +75,7 @@ class Land_Generator{
     unsigned int current_region;
     unsigned int current_sub_region;
 
-    Json::Value data; //this should be a structure
+    Json::Value data_; //this should be a structure
```

Comment:
As long as you verify the JSON its fine.

Reply:
what does that mean? like test if everything works?